### ISSUE TITLE
resizable-handle has no tabIndex and does not meet a11y/accessibility practices.

### ISSUE THREAD

**Issue by cozuya** [STRML/react-grid-layout#936]
Hi, it doesn't appear that a user can resize an element using the keyboard - you should be able to tab to the resize handle, hit space or enter to activate it, and then use the arrow keys to resize.  Has anyone managed to make this work/pass accessibility?  Thanks.

**Comment by dmitrif** [comment]
Not stale.
